client/components/chart/bar.jsx

```diff
@@ -130,20 +155,9 @@ export default class extends React.Component {
 				</li>
 			);
 		} );
+	}
 
-		return (
-			<Tooltip
-				className="chart__tooltip"
-				id="popover__chart-bar"
-				showDelay={ 200 }
-				context={ this.refs && this.refs.valueBar }
-				isVisible={ this.state.showPopover }
-				position={ this.props.tooltipPosition }
-			>
-				<ul>{ listItemElements }</ul>
-			</Tooltip>
-		);
-	};
+	setRef = ref => ( this.bar = ref );
```

Comment:
Could `this.bar` have a better name? Like `this.valueBarRef`?

Reply:
It could, but I don't think it's a bad name. In `Chart`, we use `this.chart` as well and since this component/file is called Bar, I think it's okay to call it `this.bar`.